### PR TITLE
wsproxy: port to jbuilder

### PR DESCRIPTION
**Pull request by mseri**
This will additionally require a change to the specfile when released (more details on wsproxy repo).
Depends on https://github.com/xapi-project/wsproxy/pull/5 

Signed-off-by: Marcello Seri <marcello.seri@citrix.com>

### CODE REVIEW

File: packages/xs-extra/wsproxy.master/opam [hunk 1]
```diff
@@ -7,23 +7,8 @@ license: "LGPL-2.0 with OCaml linking exception"
 homepage: "https://github.com/xapi-project/wsproxy"
 dev-repo: "https://github.com/xapi-project/wsproxy.git"
 bug-reports: "https://github.com/xapi-project/wsproxy/issues"
```
Comment (gaborigloi) [inline packages/xs-extra/wsproxy.master/opam]: Same comment about the `tags: [ "org:xapi-project" ]` tag, to make the opam files match.

File: packages/xs-extra/wsproxy.master/opam [hunk 2]
```diff
@@ -7,23 +7,8 @@ license: "LGPL-2.0 with OCaml linking exception"
...
-build: [
```
Comment (gaborigloi) [inline packages/xs-extra/wsproxy.master/opam]: Is the `build:` field deliberately omitted from the new `opam` file?

Comment (mseri) [inline packages/xs-extra/wsproxy.master/opam]: No it's a rebase conflict error :-) I need to add it back

Comment (mseri) [inline packages/xs-extra/wsproxy.master/opam]: Done

File: packages/xs-extra/wsproxy.master/opam (hunk 1 above)
Comment (mseri) [inline packages/xs-extra/wsproxy.master/opam]: Done

**Pull request merged**